docs/zh_CN/minio-client-complete-guide.md

```diff
@@ -360,7 +360,7 @@ FLAGS:
   --incomplete, -I		删除未完整上传的对象。
   --fake			   模拟一个假的删除操作。
   --stdin			   从STDIN中读对象列表。
-  --older-than value               删除N天前的对象（默认是0天）。
+  --older-than value        删除超过L天的对象M小时和N分钟LNM [d | h | m]。（默认是0天）。
```

Comment:
I think we don't need to modify the Chinese docs for the PR. There is a community person who does this for us :) . AFAIK